javascript/spread-operator.md

```diff
@@ -0,0 +1,29 @@
+### The spread operator
+
+Today I learned about a powerful feature of ES6 (ES6 being the latest standard of JavaScript implementation in the browsers), the so-called spread operator that looks like this `...`.
+
+This operator is used as a prefix for a parameter name or an element in an array. It allows for an expression to be expanded in places where multiple arguments (for function calls) or multiple elements (for array literal notation) are expected.
```

Comment:
"Commonly used as a prefix for parameter names or elements in an array.  Expressions may be expanded where multiple arguments, for function calls, or multiple elements, for array literal notation, are expected.